bin/test_travis.sh

```diff
@@ -7,6 +7,7 @@ set -x
 
 if [[ "${TEST_FLAKE8}" == "true" ]]; then
     flake8 sympy;
+    flake8 --doctests --per-file-ignores='sympy/interactive/session.py:F821' sympy;
```

Comment:
Does this run the normal flake8 checking twice?

Reply:
Yes, I found no way for flake to simoultaneously check for docs and code

Reply:
I'm not sure. It seems so, but I can't find anything in the `flake8` documentation that says whether or not the code is also checked with the `--doctests` flag.

edit: I found a reference here: https://bugs.launchpad.net/pyflakes/+bug/1223150 so it seems that the normal checking truly is done twice.